modules/openapi-generator/src/main/resources/python/model_utils.mustache

```diff
@@ -1369,14 +1370,17 @@ def model_to_dict(model_instance, serialize=True):
                            res.append(v)
                        elif isinstance(v, ModelSimple):
                            res.append(v.value)
+                       elif isinstance(v, dict):
+                           res.append(dict(map(
+                               extract_item,
+                               v.items()
+                           )))
```

Comment:
Can you please include tests?
It's not clear to me how this code can take the mapped tuple of (key, value) and generating a value based on value only can then make a dict of that data.
Tests will verify that this is working correctly for the dict and list use cases.

Reply:
those code copy from line:1381 . i only do define a var for lambda expression.